recipe/meta.yaml

```diff
@@ -1,48 +1,63 @@
-{% set version = "0.8.0" %}
+{% set version = "0.16.1" %}
 
 package:
   name: tensorflow-hub
   version: {{ version }}
 
 source:
-  url: https://github.com/tensorflow/hub/archive/v{{ version }}.tar.gz
-  sha256: 8b8d7b121855d43c52af4899552f1458ecd83bc9e50748dc4180e62e1d2bedd9
+  url: https://pypi.org/packages/py2.py3/t/tensorflow-hub/tensorflow_hub-{{ version }}-py2.py3-none-any.whl
+  sha256: e10c184b3d08daeafada11ffea2dd46781725b6bef01fad1f74d6634ad05311f
 
 build:
   number: 0
-  # This is a noarch: python package, it contains no compiled code but a
-  # compiler is required to convert protocol buffer definitions.
-  # The build has only been tested on linux-64
-  noarch: python
-  skip: true  # [not linux]
+  # tf-keras is not available for osx-arm64 because of a failed import error.
+  # no tensorflow for python 3.13
+  skip: true  # [py<39 or py>=313 or (osx and x86_64)]
   ignore_run_exports:
     - libgcc-ng
     - libstdcxx-ng
+  script: {{ PYTHON }} -m pip install -vv --no-deps --no-build-isolation tensorflow_hub-{{ version }}-py2.py3-none-any.whl
 
 requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - bazel >=0.24
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
     - numpy >=1.12.0
-    - six >=1.12.0
-    - protobuf >=3.8.0
-    - tensorflow >=1.14.0
+    - protobuf >=3.19.6
+    - tf-keras >=2.14.1
+    - tensorflow >=1.15.0
```

Comment:
conda-forge doesn't include `tensorflow` at runtime https://github.com/conda-forge/tensorflow-hub-feedstock/blob/main/recipe/meta.yaml#L23-L25.
The upstream source doesn't mention too https://github.com/tensorflow/hub/blob/v0.16.1/tensorflow_hub/pip_package/setup.py#L28-L31 .
But it's used here https://github.com/tensorflow/hub/blob/v0.16.1/tensorflow_hub/file_utils.py#L21.
Can you leave a comment?

Reply:
I will leave a comment. Thanks!